tests/models.py

```diff
@@ -0,0 +1,9 @@
+from django_extensions.db.models import TitleDescriptionModel
+
+
+class Subject(TitleDescriptionModel):
+    pass
+
+
+class Group(TitleDescriptionModel):
+    pass
```

Comment:
Added both empty models in order to emulate the external models needed for scan.